backend/src/main/java/com/empire/World.java

```diff
@@ -1479,6 +1456,10 @@ void joinBattles() {
 					}
 				}
 			}
+			for (String k : kingdoms.keySet()) {
+				if (casualtiesCaused.getOrDefault(k, 0.0) > Constants.gloryCasualtyRewardThreshold || casualtiesSuffered.getOrDefault(k, 0.0) > Constants.gloryCasualtyRewardThreshold) score(k, NationData.ScoreProfile.GLORY, 1);
```

Comment:
Maybe. The thing is the call sites into such an object would still be scattered, so I'm not convinced it would add much that NationData having a score method doesn't already. It could centralize some of the math and such (e.g. scoreGlory(double casualtiesCaused)), but the benefit is unclear to me, at least at this time.